### ISSUE TITLE
Sorting on tuples

### ISSUE THREAD

**Issue by jouvelot** [math-comp/math-comp#718]
Hello all,

While working on tuples, Emilio and I found that the following lines of code could be a useful addition to the tuple.v file.

```
Lemma sort_tupleP T n r (t : n.-tuple T): size (sort r t) == n.
Proof. by rewrite size_sort size_tuple. Qed.
Canonical sort_tuple T n r t := Tuple (@sort_tupleP T n r t).
````
Thanks.

Bye,

Pierre

**Comment by CohenCyril** [comment]
Thanks for reporting this! Yes I noticed this at some point too and forgot to do something about it. Please do not hesitate to submit a PR.

**Comment by CohenCyril** [comment]
closed by #720
